grunt/service-worker.js

```diff
@@ -4,30 +4,31 @@ const path = require('path');
 function registerServiceWorkerTasks(grunt) {
   grunt.registerMultiTask('generate-service-worker', function() {
     const done = this.async();
-    const { staticDirectoryPath, rootUrl, scriptOutputPath } = this.data;
-    writeServiceWorkerFile(staticDirectoryPath, rootUrl, scriptOutputPath)
+    const { staticDirectoryPath, scriptOutputPath } = this.data;
+    writeServiceWorkerFile(staticDirectoryPath, scriptOutputPath)
       .then(done)
+      .then(grunt.task.run('exec:cat-generated-service-worker'))
       .catch(error => {
         grunt.fail.warn(error);
         done();
       });
   });
-};
+}
 
 // Use the swPrecache library to generate a service-worker script
-function writeServiceWorkerFile(staticDirectoryPath, rootUrl, outputPath) {
+function writeServiceWorkerFile(staticDirectoryPath, outputPath) {
   const config = {
     cacheId: 'cache',
     claimsClient: true,
     skipWaiting: true,
     directoryIndex: false,
-    handleFetch: true,
+    handleFetch: false,
```

Comment:
Put a comment here to explain that we're including a custom `fetch` handler instead, otherwise it looks like the cache is disabled.